frontend/public/models/yaml-templates.ts

```diff
@@ -201,8 +201,8 @@ metadata:
   namespace: openshift-metering
 spec:
   query: namespace-memory-request
-  reportingStart: '2019-01-01T00:00:00Z'
-  reportingEnd: '2019-12-30T23:59:59Z'
+  reportingStart: '2020-01-01T00:00:00Z'
+  reportingEnd: '2020-12-30T23:59:59Z'
```

Comment:
Does it work to generate these values dynamically from the current date?

```suggestion
  reportingStart: '${new Date().getFullYear()}-01-01T00:00:00Z'
  reportingEnd: '${new Date().getFullYear()}-12-30T23:59:59Z'
```

Reply:
Is there a reason we don't go to 12-31?

Reply:
It does work to generate the values dynamically, but what about job security?  I updated it to 12-31 as well.

Reply:
> It does work to generate the values dynamically, but what about job security?

🤔 hm, good point! i didn't consider that